tests/metal/texture.slang

```diff
@@ -450,10 +450,10 @@ bool TEST_texture_float()
         // METALLIB: call {{.*}}.sample_texture_2d_array.v4f32(
         && all(Tv(1) == t2DArray.SampleLevel(samplerState, float3(u, u, 0), 0, int2(1, 1)))
 
+#ifndef EXCLUDE_DEPTH_TEXTURE // Our vulkan backend don't support SampleCmp from a rgb texture.
```

Comment:
It is because our test infra is creating an input texture of rgba format without the depth_sample capability flag and bind it to the pipeline. This leads to a Vulkan validation error in debug build. Until we enhance our test infra to be able to specify depth textures, we can't run the test.

Reply:
Oh, I see. I guess it would fail for Metal too if it actually ran.